pkgs/applications/editors/emacs/elisp-packages/melpa-packages.nix

```diff
@@ -733,6 +733,9 @@ let
           })
           else super.osx-dictionary;
 
+        # https://github.com/skeeto/at-el/issues/9
```

Comment:
Upstream fixup?
https://github.com/skeeto/at-el/commit/0489e15cd0bc2fd9da56a4147f9083c6d816ebb9

Reply:
Upstrem responsed after I created this PR.  I do not have time to check and verify the fix yet.  I will do that later if no one does.

BTW, this is the only pending fix needed to be checked and verified for now.

Reply:
resolved